build/buildall

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -ex
+
+# Install all the versions we support
+mvn -U -Dbuildver=301 install -Drat.skip=true -DskipTests
```

Comment:
We either need to interpolate scala-maven-plugin outputDirectory per buildver or call `clean` in every subsequent maven  to be sure we don't mix class files from different builds. Having interpolated output directories will also enable to call these maven commands in parallel

Reply:
This should be a for loop over buildver values

Reply:
liek I mentioned in comments there is a lot of things I want to do to improve this script, I'll add clean here but otherwise can do other stuff later.